CI/include/build_support_windows_cross.sh

```diff
@@ -31,12 +31,18 @@ fi
 install_tools() {
     sudo apt ${QUIET:+--quiet} -y install automake cmake curl git libtool meson mingw-w64 mingw-w64-tools ninja-build pkg-config wget yasm
     check_and_fetch 'https://www.nasm.us/pub/nasm/releasebuilds/2.15.01/nasm-2.15.01.tar.xz' '28a50f80d2f4023e444b113e9ddc57fcec2b2f295a07ce158cf3f18740375831'
+    FILE=/usr/local/bin/nasm
```

Comment:
yeah i should have passed a which nasm; even with that fix, i'm dropping that commit because I'm too lazy to make a version check. 
I've changed the 1st commit message